doc/user/content/concepts/indexes.md

```diff
@@ -82,6 +86,28 @@ You can also explicitly specify the cluster:
 CREATE INDEX idx_on_my_view IN CLUSTER active_cluster ON my_view (...);
 ```
 
+## Usage patterns
+
+### Indexes on views vs. materialized views
```

Comment:
Oh, not at all.  The "indexes on a view" is my transitional phrasing where we elevate the prominence of indexes to be at the same level of materialized views.  This way,

- we start to get people to think of "Oh ... indexes... they store results in memory. For views, they also incrementally update ..." 
- And later on for stacked views, if we suggest indexing the query serving view instead of intermediate views, we've already introduced the separation.  

But, let me see if I can use both phrasing.